email_reminder/utils/reminder.py

```diff
@@ -10,14 +10,29 @@ def fetch_reminder_doctypes():
 
 @frappe.whitelist()
 def send_email(message, recipients, reminder_schedule_date, doctype, docname, site):
-    if not( message and recipients and reminder_schedule_date):
-        frappe.throw(_("Please Check if All Feilds in Messaging is Full !!"))
+    
+    error_message = []
+    hasError = False
+    if not message:
+        error_message.append(f'<li>{_("Message")}</li>')
+        hasError = True
 
+    if not reminder_schedule_date:
+        error_message.append(f'<li>{_("Reminder Date and Time")}</li>')
+        hasError = True
     try:
-        data = json.loads(recipients)
-        emails = [email["email"] for email in data if email]
+        if isinstance(recipients, str):
+            emails = json.loads(recipients)
+        elif isinstance(recipients, list):
+            emails = recipients
+        
         if not emails:
-            frappe.throw(_("Please add at least one email"))
+            error_message.append(f'<li>{_("Recipients Email")}</li>')
+            hasError = True
+
+        if hasError:
+            frappe.msgprint(f'{_("Mandatory fields required for Email Reminder")}<ul>{"".join(error_message)}</ul>')
```

Comment:
How did the program stop the process from continuing when you did not use any type of throw execution?

Reply:
there is `return "Error No Log"` if there is no emails, message..